image/init.sh

```diff
@@ -20,6 +20,13 @@ fi
 
 trap stop_all HUP INT QUIT KILL TERM
 
+
+# initialize the db (pwd: admin) and create users
+sleep 5  # wait for PG to start...
```

Comment:
I'd still add a check if this is actually up after 5sec. so I'd go with this proposed solution:
```bash
while ! curl http://$POSTGRES_PORT_5432_TCP_ADDR:$POSTGRES_PORT_5432_TCP_PORT/ 2>&1 | grep '52'
do
  sleep 1
done
```
it's not big overkill and will give you confidence it's up at this point.